src/snowflake/cli/app/commands_registration/builtin_plugins.py

```diff
@@ -20,3 +21,6 @@
     "streamlit": streamlit_plugin_spec,
     "git": git_plugin_spec,
 }
+
+if FeatureFlag.ENABLE_SNOWGIT.is_disabled():
+    del builtin_plugin_name_to_plugin_spec["git"]
```

Comment:
Do this the other way around - if enabled, add it to the dict

Reply:
@sfc-gh-turbaszek done

@sfc-gh-pjob discussed on zoom